feat : 싱글 룸 생성 [#25]

seoo890 opened the pull request:
- Math.random() 함수를 이용해 난수를 방 아이디로 하는 싱글 방 생성 후 요청한 socket이 바로 입장하도록 설정
- 추후 문제출제 기능 구현할 때 난수 방 아이디가 문제가 될 시 fix할 예정
- single의 네임스페이스 이름은 single, url도 /single로 설정함

Commit:
- Math.random() 함수를 이용해 난수를 방 아이디로 하는 싱글 방 생성 후 요청한 socket이 바로 입장하도록 설정
- 추후 문제출제 기능 구현할 때 난수 방 아이디가 문제가 될 시 fix할 예정
- single의 네임스페이스 이름은 single, url도 /single로 설정함.

Commit:
Revert "- Math.random() 함수를 이용해 난수를 방 아이디로 하는 싱글 방 생성 후 요청한 socket이 바로 입장하도록 설정"

This reverts commit 06c40bbc903bbf9240dc2bee7a31e12c9ca57969.

Commit:
feat : 싱글 룸 생성 [#25]

- Math.random() 함수를 이용해 난수를 방 아이디로 하는 싱글 방 생성 후 요청한 socket이 바로 입장하도록 설정
- 추후 문제출제 기능 구현할 때 난수 방 아이디가 문제가 될 시 fix할 예정
- single의 네임스페이스 이름은 single, url도 /single로 설정함

Commit:
Revert "- Math.random() 함수를 이용해 난수를 방 아이디로 하는 싱글 방 생성 후 요청한 socket이 바로 입장하도록 설정"

This reverts commit 06c40bbc903bbf9240dc2bee7a31e12c9ca57969.

kimhanui commented on socket.js:
```diff
@@ -5,9 +5,28 @@ module.exports = (server, app, sessionMiddleware) => {
+    socket.on('disconnect',()=>{
+      socket.leave(singleRoomId);
+    });
+    
```
빈 줄은 없게 스타일 다듬어주세요!

kimhanui reviewed (commented):
커밋 메시지에 제목이 빠졌네요 제목 추가해주세요

Commit:
feat : 싱글 룸 생성 [#25]

- Math.random() 함수를 이용해 난수를 방 아이디로 하는 싱글 방 생성 후 요청한 socket이 바로 입장하도록 설정
- 추후 문제출제 기능 구현할 때 난수 방 아이디가 문제가 될 시 fix할 예정
- single의 네임스페이스 이름은 single, url도 /single로 설정함.

Commit:
feat : 싱글 룸 생성 [#25]

- Math.random() 함수를 이용해 난수를 방 아이디로 하는 싱글 방 생성 후 요청한 socket이 바로 입장하도록 설정
- 추후 문제출제 기능 구현할 때 난수 방 아이디가 문제가 될 시 fix할 예정
- single의 네임스페이스 이름은 single, url도 /single로 설정함.

Commit:
chore : 공백 제거

- 요청받은 불필요한 공백 제거
